### PR TITLE
Correct linter error.

### PR DESCRIPTION
**Pull request by worldomonation**
#### Changes proposed in this Pull Request

* do not introduce section with emphasis.

### CODE REVIEW

**Comment by worldomonation** [comment]
Sorry for causing the linter error, better to wait for a review before merging even something like documentation updates.

**Review by scinos — APPROVED** [review]
Thanks for fixing it!

**Pull request merged**
